src/protocol/modulus_conversion/specialized_mul.rs

```diff
@@ -359,7 +359,7 @@ pub mod tests {
 
             let a_shares = share(a, &mut rng);
 
-            let record_id = RecordId::from(0_u32);
+            let record_id = RecordId::from(0);
 
             let iteration = format!("{}", i);
```

Comment:
Can you hold off on landing this until we do #235 ?  This code here needs to stop using `RecordId::from(0)` and putting the value of `i` into a string.